lib/install/management/dispatcher.go

```diff
@@ -125,12 +131,60 @@ func (d *Dispatcher) initDiagnosticLogs(conf *metadata.VirtualContainerHostConfi
 	}
 }
 
+func (d *Dispatcher) registerExtension(conf *metadata.VirtualContainerHostConfigSpec) error {
+	log.Infoln("Registering VCH as a vSphere extension")
+	extensionManager := object.NewExtensionManager(d.session.Vim25())
+	extension := types.Extension{
+		Description: &types.Description{
+			Label:   "VIC",
+			Summary: "vSphere Integrated Containers Virtual Container Host",
+		},
+		Company: "VMware, Inc.",
+		Version: "0.2",
+		Key:     "com.vmware.vic",
+	}
```

Comment:
oh ha the 0.2 is nonsense isn't it, my bad, that's from where I was looking at some of Doug's Bonneville code and for some reason didn't change it. Do we have a version constant living somewhere?